config/zones.json

```diff
@@ -1524,12 +1524,49 @@
     ]
   },
   "US-NY": {
+    "capacity": {
+      "coal": 1113,
+      "gas": 11686,
+      "hydro": 3538,
+      "hydro storage": 1297,
+      "nuclear": 5292,
+      "oil": 8260,
+      "solar": 250,
+      "wind": 1010,
+      "biomass": 50
```

Comment:
👍

Reply:
this is for NY - it was accepted into the master?

Reply:
not yet I think